### ISSUE TITLE
Django admin page raise TypeError

### ISSUE THREAD

**Issue by liangliangyy** [timmyomahony/django-pagedown#72]
my code like this:
adminform.py:
```python
class ArticleForm(forms.ModelForm):
    body = forms.CharField(widget=AdminPagedownWidget())

    class Meta:
        model = Article
        fields = '__all__'
```
admin.py:
```python
class ArticlelAdmin(admin.ModelAdmin):
    search_fields = ('body',)
    form = ArticleForm
....others
```
Django Version:2.1
django-pagedown version:1.0.5
at admin page Django will raise TypeError:
```
render() got an unexpected keyword argument 'renderer'
```

**Comment by liangliangyy** [comment]
When is this version updated?

**Comment by timmyomahony** [comment]
Should be fixed as of the latest release `1.0.6`
